src/services/edusharing/services/EduSharingConnector.js

```diff
@@ -81,7 +81,12 @@ class EduSharingConnector {
 				throw Error('authentication error with edu sharing');
 			}
 
-			return result.headers['set-cookie'][0];
+			for (const cookie of result.headers['set-cookie']) {
+				if (cookie.startsWith('JSESSIONID')) {
```

Comment:
it is always JSESSIONID. be explicit here

Reply:
The EduSharingConnector will be merged from main after PR https://github.com/hpi-schul-cloud/schulcloud-server/pull/3810 is done. So no review needed at this point in first place. Sorry for the trouble.